ironfish/src/account/accounts.ts

```diff
@@ -630,10 +642,21 @@ export class Accounts {
         }
 
         if (!map.spent) {
+          let confirmedMinimumBlocks = false
+
+          const { blockHash } = transactionMapValue
+          if (blockHash) {
+            const block = await this.chain.getBlock(Buffer.from(blockHash, 'hex'))
```

Comment:
We don't use the transactions so we can just get the header here instead of the whole block.

Reply:
We're having really bad perf problems right now with getBalance(), and I think this might make it significantly worse if we're loading every block for ever note that's unspent. I think we should consider this, and test it using the bank wallet to see the difference.

Reply:
Good call, updated.